src/js/webrequest.js

```diff
@@ -738,6 +738,31 @@ function dispatcher(request, sender, sendResponse) {
     badger.storage.revertUserAction(request.origin);
     sendResponse();
 
+  } else if (request.type == "downloadWhitelist") {
+    chrome.storage.sync.get("disabledSites", function (store) {
+      if (store.hasOwnProperty("disabledSites")) {
+        var settings = badger.getSettings();
```

Comment:
Could we use `let` or `const` (when appropriate) instead of `var`? I know we have `var` all over the place right now; no need to fix existing usage, but let's switch to `let`/`const` going forward.